src/components/check-filter.module.css

```
@@ -42,6 +42,11 @@
   justify-content: space-between;
 }
 
+.filter-options {
+  height: 200px;
```

```suggestion
  max-height: 200px;
```

I *think* using max-height here would keep this from taking up blank space when the brands list is short - i.e. when using a different dataset

---

oh good call